### PR TITLE
CALS 5531- Using Backspace After Selection in Drop Down Menu Returns User to Previous Screen

### PR DESCRIPTION
**Pull request by Nasir2712**
#### Which User story does this relate to?
-https://osi-cwds.atlassian.net/browse/CALS-5531

#### Brief feature description
- fix backspace navigation on radio, checkbox and button.

#### Does this PR block/effect other stories?
-

#### Do any commands needs to be run in order to run the project with this PR?
-

#### Code coverage percent diff as a result of this PR?
-

### CODE REVIEW

File: app/javascript/rfa_forms/rfa01a_edit_view/applicantCardsGroup.jsx [hunk 1]
```diff
@@ -82,7 +82,7 @@ export default class ApplicantCardsGroup extends React.Component {
         </div>
         <div className='add-another col-xs-12 col-sm-12 col-md-12 col-lg-12'>
           <div className='text-center'>
-            <button onClick={this.addCard} className='btn btn-default'>Add Another Applicant +</button>
+            <button onClick={this.addCard} id='addAnotherApplicant' className='btn btn-default'>Add Another Applicant +</button>
```
Comment (lauherkcambria) [inline app/javascript/rfa_forms/rfa01a_edit_view/applicantCardsGroup.jsx]: this looks like a good opportunity to reuse the button component

File: app/javascript/packs/application.js [hunk 2]
```diff
@@ -21,8 +21,8 @@ ReactRailsUJS.useContext(componentRequireContext)
 document.addEventListener('keydown', keyDownSelect)
 function keyDownSelect (e) {
   let keyCode = e.which
-  if (document.activeElement === document.body || document.activeElement.tagName === 'SELECT' || document.activeElement.type === 'checkbox') {
-    if (e.keyCode === 32 || e.keyCode === 8) {
+  if (document.activeElement === document.body || document.activeElement.tagName === 'SELECT' || document.activeElement.tagName === 'BUTTON' || document.activeElement.type === 'checkbox' || document.activeElement.type === 'radio') {
```
Comment (ratneshraval) [inline app/javascript/packs/application.js]: Can you format this little better with each condition on a new line, so it is more readable.

**Comment by ratneshraval** [comment]
Probably need to merge, delete and recreate cassettes.

File: app/javascript/packs/application.js [hunk 3]
```diff
@@ -21,8 +21,8 @@ ReactRailsUJS.useContext(componentRequireContext)
 document.addEventListener('keydown', keyDownSelect)
 function keyDownSelect (e) {
```
Comment (lauherkcambria) [inline app/javascript/packs/application.js]: would using 'keypress' over 'keydown' offer a performance boost?

File: app/javascript/rfa_forms/rfa01a_edit_view/applicantCardsGroup.jsx (hunk 1 above)
Comment (lauherkcambria) [reply app/javascript/rfa_forms/rfa01a_edit_view/applicantCardsGroup.jsx]: we'll address this later with the proper styling, its not really  necessary as apart of this pr. thanks for taking a look at it though! 👍 😄

Comment (Nasir2712) [reply app/javascript/rfa_forms/rfa01a_edit_view/applicantCardsGroup.jsx]: thanks for commenting

**Pull request merged**